### PR TITLE
[TIMOB-23313] (6_0_X) Disabled Swift settings for main Titanium project

### PR DESCRIPTION
**Pull request by cb1kenobi**
[TIMOB-23313] Disabled Swift settings for main Titanium project, but ensure they are enabled in the extension targets.

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23313

### CODE REVIEW

**Comment by feons** [comment]
*LGTM.*

File: iphone/iphone/Titanium.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2947,14 +2947,14 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 241EAEC5118E2BA90081A5BE /* project.xcconfig */;
 			buildSettings = {
-				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
 				ALWAYS_SEARCH_USER_PATHS = NO;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				CODE_SIGN_ENTITLEMENTS = Titanium.entitlements;
```
Comment (hansemannn) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: I notived that this is called `"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";` on 6.1.x and master.

File: iphone/iphone/Titanium.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2982,13 +2982,14 @@
...
 				COPY_PHASE_STRIP = YES;
-				EMBEDDED_CONTENT_CONTAINS_SWIFT = YES;
+				DEVELOPMENT_TEAM = "";
```
Comment (hansemannn) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: This is also not included in 6.1.x and master.

Comment (cb1kenobi) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: `DEVELOPMENT_TEAM` was removed in master when master was 6.1.0.

File: iphone/iphone/Titanium.xcodeproj/project.pbxproj (hunk 1 above)
Comment (cb1kenobi) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: So? There was probably a reason to do it in master/6.1.x, but that has nothing to do with this PR.

Comment (hansemannn) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: Just wondered and mentioned it, eaasy man! 🙂

Comment (mukherjee2) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: Hans, Can I take this a "CR Passed?" FR passed, so may I merge it?

**Review by mukherjee2 — APPROVED** [review]
FR passed with this environment:
NPM Version: 3.10.10
Node Version: 6.10.1
Mac OS: 10.12.3
Appc CLI: 6.1.0
Appc CLI NPM: 4.2.8
Appcelerator Studio, build: 4.8.1.201612050850
Xcode 8.2.1

I followed the test steps outlined in the JIRA ticket and found that only the Watch targets had Swift enabled in the Xcode project.

**Review by hansemannn — APPROVED** [review]

File: iphone/iphone/Titanium.xcodeproj/project.pbxproj (hunk 1 above)
Comment (hansemannn) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: Yeah, we're ready to merge. I'll go ahead and merge all three then, thx!

**Pull request merged**